.github/workflows/community-tutorial-checks.js.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 The Google Earth Engine Community Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow will do a clean install of node and run tests.
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Community Tutorial checks
+
+on:
+  push:
+    branches: [master]
+    paths: ["tutorials/**"]
```

Comment:
Similar to above: if this is for JS tutorials, WDYT about restricting the search to `**.js`
to be more clear/explicit about what is what is being targeted for checks.

Reply:
See above.

Reply:
Ack